devel/arduino-cli/Portfile

```diff
@@ -3,8 +3,10 @@
 PortSystem          1.0
 PortGroup           golang 1.0
 
-go.setup            github.com/arduino/arduino-cli 0.18.3
-revision            1
+# commit with fix for the golang.org/x/sys package
+go.setup            github.com/arduino/arduino-cli dedb99b8a5fd76fd08e0b1054a62015eae80727e
```

Comment:
is this the only commit after the 0.19 release (sorry am my phone and its not as easy to check quickly). If so, this could be fine but otherwise I think the preferred way here would be to grab the released version and add a patchfile with this commit.

Reply:
There are three more commits between the tag and the commit with the fix. Usually, I ignore it just create a patch from the commit or PR. But in this case, the previous commits make changes to those same files.

```console
$ git log --oneline
...
dedb99b8 [skip-changelog] Bump go-serial to v1.3.2 (#1427)
9133101e [skip changelog] Update cobra to latest version (#1430)
f4f77c03 Revert commits 3476b3ae and ef359508
ef359508 go mod tidy
3476b3ae bump github.com/spf13/cobra to 1.2.1
...
```